lib/Elevate/Blockers/MountPoints.pm

```diff
@@ -15,10 +15,53 @@ use cPstrict;
 use parent qw{Elevate::Blockers::Base};
 
 use constant FINDMNT_BIN => '/usr/bin/findmnt';
+use constant MOUNT_BIN   => '/usr/bin/mount';
 
 use Log::Log4perl qw(:easy);
 
 sub check ($self) {
+    $self->_check_for_rhel_23449();
+    $self->_ensure_mount_dash_a_succeeds();
+    return;
+}
+
+sub _ensure_mount_dash_a_succeeds ($self) {
+
+    # Only do this in start mode because it can change the file system mounts
+    return if $self->is_check_mode();
```

Comment:
Is there a reason we don't want to do this in check? Maybe we discussed it and I've slept since then.

Reply:
Never mind